src/neuigkeiten/2024-01-06-zeiterfassung-2.0.0/index.md

```diff
@@ -0,0 +1,105 @@
+---
+draft: true
+layout: layout-neuigkeiten
+date: 2024-01-06
+title: Arbeitszeiten, Feiertagsregelung und mehr Berechtigungen in der Zeiterfassung
+author: Benjamin Seber
+release-version: 2.0.0
+teaserImage: /static/zeiterfassung-multi-device.png
+description: Kroatische Feiertage für eine einfache Urlaubsplanung
+---
+
+Zum Jahresstart gibt es spannende Aktualisierungen bei der Zeiterfassung:
+
+- flexible Arbeitszeiten für Vertragsänderungen
+- Bundesländer und Feiertagsregelungen individuell anpassen
+- Berechtigungen direkt in der Anwendung vergeben
+
+<!-- more -->
+
+## Flexible Arbeitszeiten abbilden
+
+Ab sofort könnt ihr mühelos mehrere Arbeitszeiten erstellen und verwalten. Sei es aufgrund von Teilzeitvereinbarungen,
+Vollzeitbeschäftigungen oder anderen Vertragsänderungen. Die Anwendung gibt euch auf der Personenübersichtseite eine
+kompakte Ansicht damit ihr stets den Überblick behaltet.
```

Comment:
Hier könnte man noch einen Satz ergänzen, das das ab einem definierbaren Zeitpunkt (in der Zukunft?) möglich ist. Also man muss es nicht am Tag der Änderung pflegen, sondern kann es bequem bereits im Voraus pflegen, sobald die Änderung beschlossen ist